### ISSUE TITLE
Repeated mutation only shared first time

### ISSUE THREAD

**Issue by leonardpauli** [xanf/vuex-shared-mutations#2]
The "storage" event seems to be fired only when localStorage is changed (tested in Google Chrome 56), that is, not if the same change is made twice:

```
// In tab #1
window.addEventListener('storage', console.log)

// In tab #2
localStorage.x = 'a'
localStorage.x = 'b'
localStorage.x = 'a'
localStorage.x = 'a'

// In tab #1
// console.log only fired 3 times, not 4
```

Because this plugin doesn't reset the the value between setting mutation, the [counter example](https://github.com/vuejs/vuex/tree/dev/examples/counter) (after adding this plugin) wouldn't sync as expected (the same "increment" mutation would be committed multiple times, but only the first one is shared).

To fix this, reset the value after setting it (`window.localStorage.removeItem(key);` in `store.subscribe`) and abort in the event listener (`if (event.newValue === null) return;`)

**Comment by xanf** [comment]
Thank you for your awesome input

**Comment by xanf** [comment]
Fix released in v0.0.3
